File: ssm-chaincode/ssm-chaincode-client/src/test/java/ssm/chaincode/client/SsmClientTestBuilder.java
package ssm.chaincode.client;

import java.io.IOException;

public class SsmClientTestBuilder {

    public static SsmClient build() throws IOException {
        String configFileName = getConfigFileName();
        return SsmClientBuilder.builder(configFileName).withChannelId("sandbox").withSsmId("ssm").build();
    }

    private static String getConfigFileName() {
        String profile = System.getenv("PROFILES_ACTIVE");
        System.out.println("////////////////////////////");
        System.out.println(profile);
        System.out.println("////////////////////////////");
        if("gitlab".equalsIgnoreCase(profile)) {
            return "ssm-client-gitlab.properties";
        }
        return "ssm-client.properties";
    }
}

File: ssm-chaincode/ssm-chaincode-client/src/test/kotlin/ssm/chaincode/client/SsmClientTestBuilder.kt
package ssm.chaincode.client

import java.io.IOException
import ssm.chaincode.client.SsmClientBuilder.Companion.builder

object SsmClientTestBuilder {
	@Throws(IOException::class)
	fun build(): SsmClient {
		val configFileName = configFileName
		return builder(configFileName).withChannelId("sandbox").withSsmId("ssm").build()
	}

	private val configFileName: String
		private get() {
			val profile = System.getenv("PROFILES_ACTIVE")
			println("////////////////////////////")
			println(profile)
			println("////////////////////////////")
			return if ("gitlab".equals(profile, ignoreCase = true)) {
				"ssm-client-gitlab.properties"
			} else "ssm-client.properties"
		}
}